src/init.ts

```diff
@@ -405,6 +385,26 @@ export class NightwatchInit {
     this.otherInfo.tsOutDir = '';
   }
 
+  setupComponentTesting(answers: ConfigGeneratorAnswers) {
+    if (answers.uiFramework === 'react' || answers.uiFramework === 'vue') {
```

Comment:
Why checking for 'vue' here when the file is not being copied in case of vue?

Reply:
Talked to Harshi, this was left out on his side and should be removed.

Reply:
why remove vue?

Reply:
'vue' was being checked here because earlier we were also creating a `vite.config.js` file, but now we are not doing that.

https://github.com/nightwatchjs/create-nightwatch/pull/77/commits/fd2204fcecaa50055b9e2af8e35286e26fc2d613

Reply:
the vue plugin is not dependent of the `vite.config.js` file. It should still be kept in the list. You can install the vue plugin without having a vite.config.js file. The plugin uses an existing config file only if it's found in the project, otherwise it uses a default config.

Reply:
Yes, but I haven't removed vue from the list. It's still there.

I have just removed vue from the following function (it was being checked earlier alongside react in the second line, but there's no need to do that since the following steps are only applicable for react):

<img width="663" alt="image" src="https://user-images.githubusercontent.com/39924567/209171905-82f9261b-2a0b-4785-bcaf-415e79831162.png">

(Sorry for the confusing commit message below related to the removal of vue.)

Reply:
Cool, thanks for the clarification.